tests/test_data_base.py

```diff
@@ -0,0 +1,77 @@
+import psycopg2
+import pytest
+from psycopg2 import sql
+
+from src.data_base import DataBase
+
+
+@pytest.fixture
+def table_name():
+    return "test_data_base"
+
+
+@pytest.fixture
+def db(table_name):
+    database = DataBase(table_name)
+    yield database
+    database.connection.close()
+
+
+@pytest.fixture
+def test_ids():
+    return "42", "43", "44"
+
+
+def test_db_context_manager(table_name):
+    with DataBase(table_name) as db:
+        assert db.table_name == table_name
+        assert isinstance(db.connection, psycopg2.extensions.connection)
+        assert isinstance(db.cursor, psycopg2.extensions.cursor)
+        assert db.cursor.closed is False
+        assert db.connection.closed == 0
+
+    assert db.cursor.closed is True
+    assert db.connection.closed == 1
+
+
+def test_add_photo(db, table_name, test_ids):
```

Comment:
Split into test_add_photo and test_add_duplicate. Testing for multiple add's seems redundant.

Reply:
done